tests/testthat/test-6-issues.R

```diff
@@ -69,3 +69,32 @@ test_that("update a secret, deleted user has no access", {
     "OpenSSL error"
   )
 })
+
+test_that("missing vault arguments", {
+  if (!(alice %in% list_users(pkg_root))){
+    add_user(alice, alice_public_key, vault = pkg_root)
+  }
+
+  if (!("secret_one" %in% list_secrets(pkg_root)$secret)){
+    add_secret(
+      "secret_one",
+      secret_to_keep,
+      users = c(alice, bob),
+      vault = pkg_root
+    )
+  }
+
+  old.o <- options(secret.vault = getOption("secret.vault", pkg_root))
```

Comment:
Please use `withr::with_options()` there instead of manually managing the options.